client/src/Models/Ringdown.js

```diff
@@ -58,14 +57,15 @@ const validatedFields = [
   'supplementalOxygenAmount',
   'temperature',
   'glasgowComaScale',
-  'all',
 ];
 
 const handleRange = (value, max = null, min = null) => {
-  if (isNaN(min)) {
-    return value && value > max ? false : true;
+  let valueToNumber = Number(value);
+  const isNumber = valueToNumber === 'number';
+  if (typeof min !== 'number') {
+    return isNumber && valueToNumber > max ? false : true;
   } else if (typeof min === 'number' && typeof max === 'number') {
-    return value && (value < min || value > max) ? false : true;
+    return typeof isNumber && (valueToNumber < min || valueToNumber > max) ? false : true;
   } else {
     return true;
   }
```

Comment:
It might make this easier to read if `isNumber` is checked only once, and the boolean expressions just evaluate to true or false, rather than getting reversed with a ternary.  

Line 64 should have a typeof in there, and none in line 68?

What if `min` is a number but `max` isn't?

Reply:
- Actually the first conditional (if typeof min !== 'number) isn't supposed to be there. Its initial purpose is being handled in the ternary operator we're working on fixing up. As to the matter of the ternary operator, I have a cleaned up solution (pushing up after resolving other comments).
- You're right, that seems to be an error on my part.
- Considering its a range, I don't think we would expect to have a min without a max or vice versa (hence why that first conditional was a bit pointless).

Reply:
Okay, if we want a range to always have both min and max, I can add a check to the FieldMetadata constructor.  I think they all do currently, anyway.